daemon/config/config_solaris.go

```diff
@@ -4,12 +4,6 @@ import (
 	"github.com/spf13/pflag"
 )
 
-var (
-	defaultPidFile = "/system/volatile/docker/docker.pid"
-	defaultGraph   = "/var/lib/docker"
-	defaultExec    = "zones"
-)
```

Comment:
I removed these because 1) they don't seem to be used anywhere, and 2) files for other operating systems seem to have been moved to the `cmd/dockerd` package while these were left behind. It looks like this should be covered by https://github.com/docker/docker/pull/28696/files#diff-adcf97f8e8b7815a2ec4660c286531ea